source/_patterns/01-molecules/teasers/teaser.yaml

```diff
@@ -85,12 +85,14 @@ properties:
         $ref: ../../01-molecules/components/meta.yaml
       formats:
         type: object
-        minProperties: 1
         properties:
-          html:
-            type: boolean
-          pdf:
-            type: boolean
+          list:
+            type: array
+            minItems: 1
+            item:
+              type: string
```

Comment:
not super future-proof if something like a description gets added (or any metadata). May be good for now given how relatively easy this schema can change.